src/buckets/components/CreateBucketDropdownItem.tsx

```diff
@@ -0,0 +1,74 @@
+// Libraries
+import React, {FC, useEffect} from 'react'
```

Comment:
this component has a lot of presumptions about the context it's initialized in, and none of that context has to do with src/buckets